src/components/chip-set/chip-set.tsx

```diff
@@ -375,9 +376,16 @@ export class ChipSet {
                     readonly={this.isFull()}
                 />
                 <div
+                    // Because MD removes some classes in chipset, we add custom
+                    // classes with similar names and expected behavior & styles.
+                    // These class names start with `lime-`, instead of `mdc-`.
```

Comment:
I don't think we really need this comment both here and in the scss, and I prefer it in the scss… what do you think @Kiarokh?

Reply:
I don't have any problems if we remove it here